.firebase/hosting.b3V0.cache

```diff
@@ -1,61 +1,47 @@
-404.html,1584738636093,103a7773023c778e55712043648742edbe0bd24d63ff726bb9383276c612307e
```

Comment:
I feel like this folder should be .gitignored

Reply:
Agreed